### ISSUE TITLE
p-chip: Missing button role on a remove icon

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#15095]
### Describe the bug

Remove icon elements are missing a button role and aria-label attribute cannot be used on a remove icon element (TimesCircleIcon & span) with no valid role attribute. 

The button role identifies an element as a button to assistive technology such as screen readers.  

### Environment

chrome

### Reproducer

https://primeng.org/chip

### Angular version

17.0.0

### PrimeNG version

17.11.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0

### Steps to reproduce the behavior

https://primeng.org/chip
